components/01-atoms/images/icons/_icon.twig

```diff
@@ -14,27 +14,12 @@
 
 {% set unique_name = icon__name ~ '-' ~ random() %}
 
-<svg {{ bem(icon__base_class, icon__modifiers, icon__blockname, icon__extra_class) }}
-
-  {% if icon__decorative %}
-    aria-hidden="true" role="img"
-  {% elseif icon__role %}
-    role="{{ icon__role }}"
-  {% endif %}
-
-  {% if icon__title %}
-    aria-labelledby="title-{{ unique_name }}"
-  {% endif %}
-
-  {% if icon__desc %}
-    aria-describedby="desc-{{ unique_name }}"
-  {% endif %}
->
-  {% if icon__title %}
-    <title id="title-{{ unique_name }}">{{ icon__title }}</title>
-  {% endif %}
-  {% if icon__desc %}
-    <desc id="desc-{{ unique_name }}">{{ icon__desc }}</desc>
-  {% endif %}
-  <use xlink:href="{{ icon__url }}icons.svg#{{ icon__name }}"></use>
+<svg {{ bem(icon__base_class, icon__modifiers, icon__blockname, icon__extra_class) }} {% if icon__decorative %} aria-hidden="true" role="img" {% elseif icon__role %} role="{{ icon__role }}" {% endif %} {% if icon__title %} aria-labelledby="title-{{ unique_name }}" {% endif %} {% if icon__desc %} aria-describedby="desc-{{ unique_name }}" {% endif %}>
+	{% if icon__title %}
+		<title id="title-{{ unique_name }}">{{ icon__title }}</title>
+	{% endif %}
+	{% if icon__desc %}
+		<desc id="desc-{{ unique_name }}">{{ icon__desc }}</desc>
+	{% endif %}
+	<use xlink:href="{{ icon__url }}icons.svg#{{ icon__name }}"></use>
```

Comment:
I think your editor might have automatically reformatted this file... This should be reverted.